src/app.jsx

```diff
@@ -1,13 +1,88 @@
-import styles from './index.scss';
+import './index.scss';
 import React from 'react';
+import pic from './weather.jpg';
 
 export default class App extends React.Component {
   render() {
+		var bgImage = {
+			backgroundImage: 'url(' + pic + ')'
+		};
+
+		var inputStyle = {
+			float: "right",
+			backgroundColor: "#ccdef9",
+			marginTop: 30,
+			marginRight: 50,
+			marginBottom: 10,
+			height: 35,
+			width: 220,
+			border: "#5092fc",
+			paddingLeft: 10
+ 		};
+
+ 		var cardsDiv = {
+ 			marginTop: 400,
+		};
+
+		var cardActive = {
+			width: 200,
+			marginLeft: 50,
+			color: "white",
+			display: "inline-block"
+		};
+
+		var tempValue = {
+			fontSize: 80,
+			marginTop: 1,
+			marginBottom: 1
+		};
+
+		var minMaxTemp = {
+			fontSize: 25,
+			marginTop: 20,
+			marginBottom: 1
+		};
+
+
+		var weatherText = {
+			fontSize: 30,
+			marginTop: 1
+		};
+
+		var iconSize = {
```

Comment:
move these rules in scss file
use - separated class/id names
use scss variables for colors etc. or anything reuseable
consult http://sass-lang.com/guide